### PR TITLE
S20 adding all picture functions jrs2

### PR DESCRIPTION
**Pull request by jsenning**
Extends Picture() constructor, adds a few Picture tests

### CODE REVIEW

File: Picture.py [hunk 1]
```diff
@@ -10,12 +10,33 @@
...
+                self.filename = self.title = arg.filename
+            except AttributeError:
+                self.filename = self.title = ''
+        elif isinstance(arg, str):
+            # arg is a string, do what JES does and make picture of string
+            self.image = PIL.Image.new("RGB", (600, 200))
+            self.extension = extension
+            self.title = arg
+            self.filename = ''
+            draw = PIL.ImageDraw.Draw(self.image)
```
Comment (Nick-Noormand) [inline Picture.py:34]: Why not just use self.addText() rather than drawing the text using PIL draw

File: Tests/test_Picture.py [hunk 2]
```diff
@@ -30,6 +31,42 @@ def makeReferenceImage(filename='refimage.jpg'):
 
 # Testing functions
 
+def test_init():
```
Comment (Nick-Noormand) [inline Tests/test_Picture.py:34]: These tests look good. Just wondering if they are working well?

File: Picture.py (hunk 1 above)
Comment (jsenning) [reply Picture.py:34]: Given that this is happening in the constructor (init function?) for the class, I don't think we have "self" defined yet.

File: Tests/test_Picture.py (hunk 2 above)
Comment (jsenning) [reply Tests/test_Picture.py:34]: They've been working for me -- do they work for you?  You'll need to install pytest...

**Pull request merged**